__tests__/app.test.js

```diff
@@ -149,4 +149,108 @@ describe("/api/articles/:article_id", () => {
         expect(body.message).toBe("Bad request");
       });
   });
+});
+describe("PATCH /api/articles/:article_id", () => {
+  test("should return specified article with updated votes when votes increased", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send({inc_votes: 23})
+      .expect(201)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(3);
+        expect(response.body.article.votes).toBe(23);
+        expect(response.body.article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("should return specified article with updated votes when votes decreased", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({inc_votes: -20})
+      .expect(201)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.votes).toBe(80);
+        expect(response.body.article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("should return specified article with unchanged votes when votes is zero", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({inc_votes: 0})
+      .expect(201)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.votes).toBe(100);
+        expect(response.body.article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("should return 404 when passed an ID that is correctly formatted but does not exist", () => {
+    return request(app)
+      .patch("/api/articles/99999")
+      .send({inc_votes: 10})
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Item not found");
+      });
+  });
+  test("should return 400 when passed an ID that is incorrectly formatted", () => {
+    return request(app)
+      .patch("/api/articles/mistake")
+      .send({inc_votes: 10})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad request");
+      });
+  });
+  test("should return 400 when passed invalid key", () => {
```

Comment:
This test is also a test for checking what happens when the 'inc_votes' property is missing. Getting a 400 is fine here, but if you wanted to patch another property in the future, then this test would cause a problem. For example if your object had an 'invalid_key' property (or any other), then you would patch that with an object where there's no 'inc_votes' property, you would receive a 400 instead of 200. 
So as I said this is ok for now, but if you wanted to future-proof this endpoint, then refactoring to 200 is recommended. (That would respond with 200 and the article unchanged.)